conjur/init/init_controller.py

```diff
@@ -45,24 +45,43 @@ def load(self):
         self.write_conjurrc()
 
         sys.stdout.write("Configuration initialized successfully!\n")
-        sys.stdout.write("To begin using the CLI, log in to the Conjur server by" \
-                         "running `conjur authn login`")
+        sys.stdout.write("To begin using the CLI, log in to the Conjur server by " \
+                         "running `conjur login`\n")
 
     def get_server_certificate(self):
         """
         Method to get the certificate from the Conjur endpoint detailed by the user
         """
         # pylint: disable=line-too-long
         if self.conjurrc_data.appliance_url is None:
-            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ")
+            self.conjurrc_data.appliance_url = input("Enter the URL of your Conjur server: ").strip()
+            if self.conjurrc_data.appliance_url is None or self.conjurrc_data.appliance_url == '':
+                # pylint: disable=raise-missing-from
+                raise RuntimeError("Error: URL is required")
+
 
         url = urlparse(self.conjurrc_data.appliance_url)
+        # TODO: Factor out the following URL validation to ConjurrcData class
+        # At this time, providing ports is not supported.
+        # All HTTPS activity will be through 443
+        if url.netloc.find(":443") != -1 or (url.scheme != '' and url.path != ''):
+            raise RuntimeError(f"Error: undefined behavior. Reason: The Conjur url format provided "
+                   f"'{self.conjurrc_data.appliance_url}' is not supported. "
+                   "Consider adding HTTPS as the prefix or remove the port if provided")
+
+        if url.scheme == '':
+            parsed_url = urllib.parse.urlparse(self.conjurrc_data.appliance_url)
+            replaced_url = parsed_url._replace(scheme="https", netloc=url.path, path='')
+            url = replaced_url
+            self.conjurrc_data.appliance_url=replaced_url.geturl()
```

Comment:
these 4 lines look very messy.
Concise and write exactly what you need.
I think you meant to add `https` if you don't have one.
urlparse might surprise you when there's no scheme.
You will have to test it very carefully, as this code can easily break.